bundles/statistics/statsgrid2016/components/IndicatorParametersList.js

```diff
@@ -39,20 +38,29 @@ Oskari.clazz.define('Oskari.statistics.statsgrid.IndicatorParametersList', funct
             event.stopPropagation();
             me.requestIndicatorSelectors();
         });
+
         return this.getElement();
     },
     setDatasets: function (datasets) {
         var me = this;
         var listEl = this.getElement().find('ul');
         listEl.empty();
         if (!datasets) {
+            this.addDatasetButton.show();
             return;
         }
+        this.addDatasetButton.hide();
```

Comment:
Why would you hide the button to add datasets if the indicator already has datasets? Indicator can have multiple datasets.